cfme/configure/configuration/__init__.py

```diff
@@ -305,219 +304,129 @@ class ServerLogDepot(Pretty):
-                "foobar",
-                "backup.acme.com",
-                username="jdoe",
-                password="xyz",
-            )
-            log_credentials.update()
-
-        """
-        pretty_attrs = ['p_type', 'name', 'uri', 'username', 'password']
-
-        server_collect_logs = Form(
-            fields=[
-                ("type", {
-                    version.LOWEST: Select("select#log_protocol"),
-                    '5.5': AngularSelect('log_protocol')}),
-                ("name", {
-                    version.LOWEST: None,
-                    "5.4": Input("depot_name")}),
-                ("uri", Input("uri")),
-                ("user", Input("log_userid")),
-                ("password", MultiFill(Input("log_password"), Input("log_verify"))),
-            ]
+    def __init__(self, p_type, depot_name=None, uri=None, username=None, password=None):
+        self.p_type = p_type
+        self.depot_name = depot_name
+        self.uri = uri
+        self.username = username
+        self.password = password
+
+        p_types = dict(
+            anon_ftp="Anonymous FTP",
+            ftp="FTP",
+            nfs="NFS",
+            smb="Samba",
+            dropbox="Red Hat Dropbox",
         )
+        self.p_type = p_types[p_type]
 
-        validate = form_buttons.FormButton("Validate the credentials by logging into the Server")
-        # add FormButton because of inconsistency in attributes for Save buttons in cfme
-        save_button = {
-            version.LOWEST: form_buttons.save,
-            "5.4": form_buttons.angular_save,
-            "5.6": form_buttons.FormButton("Save Changes", ng_click="btnClick()")
-        }
-
-        def __init__(self, p_type, name, uri, username=None, password=None):
-            assert p_type in self.p_types.keys(), "{} is not allowed as the protocol type!".format(
-                p_type)
-            self.p_type = p_type
-            self.uri = uri
-            self.username = username
-            self.password = password
-            self.name = name
-
-        @cached_property
-        def p_types(self):
-            return version.pick({
-                version.LOWEST: dict(
-                    anon_ftp="Anonymous FTP",
-                    ftp="FTP",
-                    nfs="NFS",
-                    smb="Samba",
-                    dropbox="Red Hat Dropbox",
-                ),
-                "5.5": dict(
-                    anon_ftp=sel.ByValue("Anonymous FTP"),
-                    ftp=sel.ByValue("FTP"),
-                    nfs=sel.ByValue("NFS"),
-                    smb=sel.ByValue("Samba"),
-                    dropbox=sel.ByValue("Red Hat Dropbox")
-                )
-            })
-
-        def update(self, validate=True, cancel=False):
-            """ Navigate to a correct page, change details and save.
+    def create(self, cancel=False):
+        self.clear()
+        view = navigate_to(current_appliance.server, 'DiagnosticsCollectLogsEdit')
```

Comment:
**Required**
We're trying to eliminate current_appliance use, looks like you can use NavigateToAttribute here.